src/main/java/org/generationcp/middleware/api/brapi/VariableServiceBrapi.java

```diff
@@ -12,5 +14,9 @@
 
     long countObservationVariables(VariableSearchRequestDTO requestDTO);
 
+	List<AttributeDTO> getGermplasmAttributes(AttributeSearchRequestDTO requestDTO, Pageable pageable);
```

Comment:
I think we can keep the MW service generic by accepting as parameter the list of variable types..
`List<VariableDTO> getObservationVariables(VariableSearchRequestDTO requestDTO, Pageable pageable, List<Integer> variableTypeIds);`
`long countObservationVariables(VariableSearchRequestDTO requestDTO, List<Integer> variableTypeIds);`